### PR TITLE
Verify the version past the Guava & Guice bumps

### PR DESCRIPTION
**Pull request by batmat**
Basically checking against:
* https://github.com/jenkinsci/jenkins/pull/5707
* https://github.com/jenkinsci/jenkins/pull/5858

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Appropriate autotests or explanation to why this change has no tests

### CODE REVIEW

**Comment by kuisathaverat** [comment]
@batmat Can I close this PR?

**Comment by batmat** [comment]
I actually pushed yesterday the last commit, but it seems like CI failed out of a timeout on the Windows agent precisely 60 minutes of build time would be fishy). 
This may be because of the low download rates the infra is hitting these days.
Let's retry.

**Comment by batmat** [comment]
Just resolved a conflict introduced by https://github.com/jenkinsci/saml-plugin/pull/157

File: Jenkinsfile [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
 
-buildPlugin(useContainerAgent: true, configurations: [
-  [ platform: "windows", jdk: "11" ],
-  [ platform: "linux", jdk: "11" ]
-])
\ No newline at end of file
+buildPlugin(useContainerAgent: true,
+        configurations: [
+                [platform: 'linux', jdk: '11'],
+                [platform: 'windows', jdk: '8'],
```
Comment (kuisathaverat) [inline Jenkinsfile:6]: I've recently removed the compilation with JDK 8 it is not needed

```suggestion
                [platform: 'windows', jdk: '11'],
```

File: Jenkinsfile [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+
+                // testing the Guava & Guice bumps
+                // https://github.com/jenkinsci/jenkins/pull/5707
+                // https://github.com/jenkinsci/jenkins/pull/5858
+                [ platform: "linux", jdk: "8", jenkins: '2.321', javaLevel: "8" ]
```
Comment (kuisathaverat) [inline Jenkinsfile:11]:
```suggestion
                [ platform: "linux", jdk: "11", jenkins: '2.321', javaLevel: "8" ]
```

File: Jenkinsfile (hunk 1 above)
Comment (batmat) [reply Jenkinsfile]: As you see fit.

Applied

**Review by kuisathaverat — APPROVED** [review]

**Comment by batmat** [comment]
Thanks for merging Ivan.

**Pull request merged**